app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/disposaldetails/disposal_price.scala.html

```diff
@@ -70,26 +72,23 @@
 @indirectKey = @{ if(isIndirectDisposal) ".indirect" else ""}
 
 @key = @{ "disposalPrice" }
-@title = @{ messages(s"$key$userKey$indirectKey$disposalMethodKey.title") }
 @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
+@title = @{ messages(s"$key$userKey$indirectKey$disposalMethodKey.title") }
 
-    @backLinkComponent(backLink)
+@mainTemplate(pageTitle = title, isWelshTranslationAvailable = true, withSignOutLink = true, backLinkUrl = Some(backLink.url)) {
 
     @if(hasErrors) {
         @errorSummary(form, Some(s"$key$userKey$indirectKey$disposalMethodKey"))
     }
 
-    @caption(messages("returns.disposal-details.caption"))
-
-    @formWrapper(controllers.returns.disposaldetails.routes.DisposalDetailsController.whatWasDisposalPriceSubmit(), 'novalidate -> "novalidate") {
+    @formHelper(controllers.returns.disposaldetails.routes.DisposalDetailsController.whatWasDisposalPriceSubmit(), 'novalidate -> "novalidate") {
       @unitInput(
         fieldId = key,
         value = form.data.getOrElse(key, ""),
         formError = form.error(key),
         hasErrors = form.error(key).nonEmpty,
-        label = Html(title),
+        label = pageHeading(title, Some(messages("returns.disposal-details.caption"))),
```

Comment:
Can this page use the govuk text input directly instead of the in-service custom component?